.github/fabricbot.json

```diff
@@ -5,31 +5,751 @@
 		"subCapability": "IssuesOnlyResponder",
 		"version": "1.0",
 		"config": {
-			"conditions": {
-				"operator": "and",
+		"conditions": {
+			"operator": "and",
+			"operands": [
+			{
+				"name": "labelAdded",
+				"parameters": {
+				"label": "need-info"
+				}
+			}
+			]
+		},
+		"eventType": "issue",
+		"eventNames": [
+			"issues",
+			"project_card"
```

Comment:
Do we use project_cards? Why do we list to the project_card event?  I do know that project_cards can be converted to issues and vice versa. Do we know why we did this?